### ISSUE TITLE
there is a bug when method has the keyword 'from'

### ISSUE THREAD

**Issue by Zuckonit** [timothycrosley/isort#214]
before call isort:
```
from a import d
from test import get_email_from, get_email_to
from b import c
```

after call isort:
```
from a import d
from b import c
from test import get_email_, get_email_to
```
isort version is: 3.9.4

**Comment by Zuckonit** [comment]
below function has some error:
```
    def _strip_syntax(self, import_string):
        import_string = import_string.replace("_import", "[[i]]")
        for remove_syntax in ['\\', '(', ')', ",", 'from ', 'import ']:
            import_string = import_string.replace(remove_syntax, " ")
        import_string = import_string.replace("[[i]]", "_import")
        return import_string

```

**Comment by Zuckonit** [comment]
I have gived my commit

**Comment by timothycrosley** [comment]
Hi @Zuckonit,

Thanks for catching this! I will make sure the fix is integrated before the next release.

~Timothy

**Comment by timgraham** [comment]
Pull request seems to be https://github.com/timothycrosley/isort/pull/216

**Comment by timgraham** [comment]
@Zuckonit -- I extracted the minimal bit from this PR that I needed to fix my issue (https://github.com/timothycrosley/isort/issues/224). I also added a test for at last part of your issue; see https://github.com/timothycrosley/isort/pull/230. If your other changes are necessary, I think they may belong in a separate issue. At least, we need a test to see what problem they are solving.

**Comment by timothycrosley** [comment]
This is fixed officially in release 3.9.5.

Thanks again guys for your help!

~Timothy
